packages/scripts/src/tasks/providers/subtitles.ts

```diff
@@ -0,0 +1,120 @@
+import fs from "fs";
+import * as path from "path";
+import parser from "subtitles-parser-vtt";
+import { ROOT_DIR } from "../../paths";
+import { recursiveFindByExtension, logError, logOutput, promptOptions } from "../../utils";
+import { TRANSLATIONS_CONTENT_LIST } from "../../../../app-data/translations";
```

Comment:
This import may prove problematic as we are moving app-data folders to be deployment specific. So instead I'll probably recommend importing from the root `src/assets/app_data/translations` folder instead for now, as that path will remain fixed (included in gitignore)

I'll push a quick fix for this also